rust/processor/src/utils/database.rs

```diff
@@ -46,14 +45,14 @@ pub const MAX_DIESEL_PARAM_SIZE: u16 = u16::MAX / 2;
 /// Given diesel has a limit of how many parameters can be inserted in a single operation (u16::MAX)
 /// we may need to chunk an array of items based on how many columns are in the table.
 /// This function returns boundaries of chunks in the form of (start_index, end_index)
-pub fn get_chunks(num_items_to_insert: usize, column_count: usize) -> Vec<(usize, usize)> {
-    let max_item_size = MAX_DIESEL_PARAM_SIZE as usize / column_count;
-    let mut chunk: (usize, usize) = (0, min(num_items_to_insert, max_item_size));
+pub fn get_chunks(num_items_to_insert: usize, chunk_size: usize) -> Vec<(usize, usize)> {
+    let chunk_size = min(chunk_size, MAX_DIESEL_PARAM_SIZE as usize);
```

Comment:
We probably still want to keep the default as MAX_DIESEL_PARAM_SIZE / column_count so the default size doesn't go out of bounds.